packages/web/src/components/data-entry/FormTile.jsx

```diff
@@ -517,6 +549,10 @@ const AdvancedForm = (props) => {
               isRemix={!!props.defaultFields.remix_of?.tracks?.length}
               isUpload={props.isUpload}
               initialForm={props.initialForm}
+              forceOpen={props.forceOpenAccessAndSale}
+              setForceOpen={props.setForceOpenAccessAndSale}
+              lastGateKeeper={props.lastGateKeeper}
+              setLastGateKeeper={props.setLastGateKeeper}
```

Comment:
nit: would we be able to avoid piping lastGateKeeper if we set it as a form field?

Reply:
need the same value available in both AdvancedForm and BasicForm, which are both in FormTile, which does not use the formik stuff